packages/cloud-cognitive/README.md

```diff
@@ -8,10 +8,10 @@
 > Carbon for IBM Cloud and Cognitive (@carbon/ibm-cloud-cognitive), and this
 > name can still be encountered in various places and historical logs.
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg)](#contributors-)
```

Comment:
It looks like we kind of have two variations of the `All Contributors` badge. From their [FAQs](https://allcontributors.org/docs/en/bot/faq) it looks like the following will work.
```suggestion
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-15-orange.svg?style=flat-square)](#contributors)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Reply:
So I think this version is more manual anyway since all-contributors isn’t the most monorepo friendly. (I think Matt Rosno actually opened an issue related to this in their repo.)

Reply:
Oh ok, we can revisit the `All contributors badge` then.